src/integration/integration_scheduler_scalingengine_test.go

```diff
@@ -96,7 +96,7 @@ var _ = Describe("Integration_Scheduler_ScalingEngine", func() {
 
 			Eventually(func() bool {
 				return activeScheduleExists(testAppId)
-			}, 2*time.Minute).Should(BeFalse())
+			}, 2*time.Minute, 5*time.Second).Should(BeFalse())
```

Comment:
ok cool, let's keep it at 1s and if we see performance problems at that rate maybe we should see if we can optimise the code somehow